[2023-09-11T02:53:06.309+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v02.create_postgre_table manual__2023-09-11T02:53:05.207470+00:00 [queued]>
[2023-09-11T02:53:06.314+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_postgres_operator_v02.create_postgre_table manual__2023-09-11T02:53:05.207470+00:00 [queued]>
[2023-09-11T02:53:06.315+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2023-09-11T02:53:06.322+0000] {taskinstance.py:1382} INFO - Executing <Task(PostgresOperator): create_postgre_table> on 2023-09-11 02:53:05.207470+00:00
[2023-09-11T02:53:06.326+0000] {standard_task_runner.py:57} INFO - Started process 3722 to run task
[2023-09-11T02:53:06.328+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_with_postgres_operator_v02', 'create_postgre_table', 'manual__2023-09-11T02:53:05.207470+00:00', '--job-id', '346', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgre_operator.py', '--cfg-path', '/tmp/tmptr8bw43m']
[2023-09-11T02:53:06.331+0000] {standard_task_runner.py:85} INFO - Job 346: Subtask create_postgre_table
[2023-09-11T02:53:06.373+0000] {task_command.py:415} INFO - Running <TaskInstance: dag_with_postgres_operator_v02.create_postgre_table manual__2023-09-11T02:53:05.207470+00:00 [running]> on host 2e8147b3a465
[2023-09-11T02:53:06.418+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='andy_hsu' AIRFLOW_CTX_DAG_ID='dag_with_postgres_operator_v02' AIRFLOW_CTX_TASK_ID='create_postgre_table' AIRFLOW_CTX_EXECUTION_DATE='2023-09-11T02:53:05.207470+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-11T02:53:05.207470+00:00'
[2023-09-11T02:53:06.419+0000] {sql.py:274} INFO - Executing: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
            
[2023-09-11T02:53:06.425+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-11T02:53:06.432+0000] {base.py:73} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-09-11T02:53:06.456+0000] {sql.py:418} INFO - Running statement: 
            create table if not exists dag_runs (
                dt date,
                dag_id character varying,
                primary key (dt, dag_id)
            )
            , parameters: None
[2023-09-11T02:53:06.466+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_with_postgres_operator_v02, task_id=create_postgre_table, execution_date=20230911T025305, start_date=20230911T025306, end_date=20230911T025306
[2023-09-11T02:53:06.502+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-11T02:53:06.515+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
